modules/conflict_resolver/jsx/conflictResolverIndex.js

```diff
@@ -0,0 +1,293 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'Loader';
+import {Tabs, TabPane} from 'Tabs';
+import FilterForm from 'jsx/FilterForm';
+import DynamicDataTable from 'jsx/DynamicDataTable';
+/**
+ * This file contains the React classes for conflict resolver
+ * module.
+ */
+
+/**
+ * This is the React class for Unresolved Conflicts
+ */
+class UnresolvedConflictsPane extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      Data: this.props.data,
+    };
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  updateFilterState(filter) {
+    this.setState({filter});
+  }
+
+  formatColumn(column, cell, rowData, rowHeaders) {
+    if (loris.hiddenHeaders.indexOf(column) > -1) {
+      return null;
+    }
+    // Create the mapping between rowHeaders and rowData in a row object.
+    let row = {};
+
+    rowHeaders.forEach(function(header, index) {
+      row[header] = rowData[index];
+    });
+
+    if (column === 'Correct Answer') {
+      const value1 = row.Value1;
+      const value2 = row.Value2;
+      const hash = row.Hash;
+      return <td>
+        <select name={hash} className='form-control input-sm'>
+          <option value='none'>Unresolved</option>
+          <option value='1'>{value1}</option>
+          <option value='2'>{value2}</option>
+        </select>
+      </td>;
+    }
+    return <td>{cell}</td>;
```

Comment:
i would suggest formatting the instruments column so that the value of the instrument is rendered instead of the key

Reply:
@zaliqarosli nice catch and I'm embarrassed to not have noticed this!

Reply:
yeahh ive realized recently that the react versions often have the values in the select filters but not in the table so i've only recently been changing them!